app/helpers/components_helper.rb

```diff
@@ -28,6 +28,12 @@ def mentor_solutions_list(request, sort_options)
     react_component("mentor-solutions-list", { request: request, sort_options: sort_options })
   end
 
+  def track_list(tracks, request)
```

Comment:
This is an example of us passing initial data. We can adapt this pattern for our other components.

Reply:
I don't see the `initialData` field being used anywhere? Is that correct?

Reply:
This needs to be changed when https://github.com/exercism/v3-website/pull/82 is merged.